src/controllers/commit_controller.js

```diff
@@ -0,0 +1,134 @@
+'use strict'
+
+const GitHub            = require('../models/github').GitHub
+  , date                = require('../common/Date')
+  , ghx9rc              = require('../common/gh-x9rc')
+  , StringExtensions    = require('../common/String')
+  , express             = require('express')
+
+const _createErrorJSON = (error, require_login, code = 500, message) => {
+  return JSON.stringify({ error, require_login, code, message })
+}
+
+exports.CommitController = class CommitController {
+  
+  constructor() {
+    this.github = new GitHub()
+    this.CRESCER_REPO_NAME = ghx9rc.main_repository || 'crescer-2016-1'
+    this.router = express.Router()
+    this.router.get('/', (req, res) => this.get(req, res))
+  }
+
+  // Actions
+  get(req, res) {
+    // TODO: criar método facade pra isso
+    res.writeHead(200, { 'Content-Type': 'application/json; charset=utf-8' });
+    // TODO: extrair o nome deste cookie para constante
+    var token = req.cookies['gh-x9-v2-auth'];
+    if(!!token) {
```

Comment:
Apesar da imensa discussão, não acho tão obscuro devido à natureza da linguagem (valores falsy não necessariamente são false):

http://stackoverflow.com/a/784946/5194966
https://github.com/getify/You-Dont-Know-JS/blob/master/up%20&%20going/ch2.md#truthy--falsy